libmscore/style.cpp

```diff
@@ -824,8 +824,10 @@ static const StyleType styleTypes[] {
       { Sid::measureNumberFontSpatiumDependent, "measureNumberFontSpatiumDependent", true },
       { Sid::measureNumberFontStyle,        "measureNumberFontStyle",       int(FontStyle::Normal) },
       { Sid::measureNumberColor,            "measureNumberColor",           QColor(0, 0, 0, 255) },
-      { Sid::measureNumberOffset,           "measureNumberOffset",          QPointF(0.0, -2.0) },
+      { Sid::measureNumberOffset,           "measureNumberOffset",          QPointF(0.0, 0.0) },
```

Comment:
Sure about this change?

Reply:
Having it here was fine when mn could only be above the staff, but didn't work when they got under. I had to move this 2sp of margin [here](https://github.com/musescore/MuseScore/pull/5817/files#diff-c6d784b581b8be852aa62d1fc8e37770R138)

Reply:
While I agree with the idea behind this change, it unfortunately breaks existing scores that had custom style style settings based on the old interpretation - their measure numbers are now 2 sp too high.  See https://musescore.org/en/node/306603

Reply:
Oh, bad regression, I'll have a look at it.